src/modules/pmc/src/lib/PmcBase.cpp

```diff
@@ -118,7 +127,7 @@ int PmcBase::GetInfo(const char* clientName, MMI_JSON_STRING* payload, int* payl
 
 int PmcBase::Set(const char* componentName, const char* objectName, const MMI_JSON_STRING payload, const int payloadSizeBytes)
 {
-    if(!CanRunOnThisPlatform())
+    if (!CanRunOnThisPlatform())
     {
         return ENODEV;
```

Comment:
>return ENODEV; [](http://example.com/codeflow?start=8&length=14)

This needs an error trace to tell the operator why this module refuses to run (here and elsewhere)

Reply:
Found it in the CanRunOnThisPlatform body. Looks good